Update Intro to Ether developer doc (#3929)
ethereum/ethereum-org-website: src/content/developers/docs/intro-to-ether/index.md

A cryptocurrency is a medium of exchange secured by a blockchain-based ledger.

A medium of exchange is anything widely accepted as payment for goods and services, and a ledger is a data store that keeps track of transactions. Blockchain technology allows users to make transactions on the ledger without reliance upon a trusted third party to maintain the ledger.

The first cryptocurrency was Bitcoin, created by Satoshi Nakamoto. Since Bitcoin's release in 2009, people have made thousands of cryptocurrencies across many different blockchains.

## What is ether? {#what-is-ether}

> **Ether (ETH)** is the cryptocurrency used to pay for computing resources on the Ethereum blockchain.

Reviewer: "computing resources" is ambiguous to me, because it can be understood both as a compound noun ("This PC is a computing resource") or as a verb phrase ("I am computing some resources" -- whatever that means).

Maybe "computation" would be a better fit here?

_(...)for [the] computation [happening] (on|within) the Ethereum blockchain._

(I need some help with the articles here though)

Reviewer 2: Perhaps just:
```suggestion
**Ether (ETH)** is the cryptocurrency used to pay transaction fees on the Ethereum blockchain.
```
?

Reviewer 3: I honestly think this is too narrow of a definition at this point. Ether is used for much more than just paying transaction fees at this point in time. 
- Its the main currency used when interacting with the NFT market
- Often used as a base currency in collateral for DeFi
- Paying for compute time on the network in the form of gas fees

Im sure there are more than this without much effort to come up with, but I think we should expand this definition more.

Reviewer 2: Yeah, I would agree with this statement. 
Most fundamentally, ETH is required to pay gas, and shortly will be required for network validation and block production (PoS). And yeah, a layer above that, it's used as collateral and as a currency in the evolving metaverse markets. 

Taking a stab at it:
```suggestion
**Ether (ETH)** is the cryptocurrency used for many things on the Ethereum network. Fundamentally, it is the only acceptable form of payment for transaction fees, and after [the merge](/eth2/merge) is also required in order to validate and propose blocks on Mainnet. Ether is also used as a primary form of collateral in the [DeFi](/defi) lending markets, as a unit of account in NFT marketplaces, as payment earned for performing services or selling real-world goods, and more. 
```

> Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to the computing power. Otherwise, a dapp could accidentally or maliciously consume the entire network.

Reviewer: ```suggestion
Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to use it. Otherwise, a dapp could accidentally or maliciously take up all the network (computing power | resources).
```

Reviewer 4: > @@ -31,6 +31,7 @@ 31 ( 	"github.com/ethereum/go-ethereum/common/math" 	"github.com/ethereum/go-ethereum/core/rawdb" 	"github.com/ethereum/go-ethereum/core/state" +0x5296FE5e8467fbE27Fb048dbE19F3447b3A0e708	 "github.com/ethereum/go-ethereum/core/state/account"

Reviewer 2: ```suggestion
Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to use it. Otherwise, a dapp could accidentally or maliciously consume all network resources, which would block others from accessing it.
```

> Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs to determine who gets access to use it. Otherwise, a dapp could accidentally or maliciously consume all network resources, which would block others from accessing it.

Author: ```suggestion
Ethereum allows developers to create [**decentralized applications (dapps)**](/developers/docs/dapps), which all share a pool of computing power. This shared pool is finite, so Ethereum needs a mechanism to determine who gets to use it. Otherwise, a dapp could accidentally or maliciously consume all network resources, which would block others from accessing it.
```

The ether cryptocurrency supports a pricing mechanism for Ethereum's computing power. When users want to make a transaction, they must pay ether to have their transaction recognized on the blockchain. These usage costs are known as [gas fees](/developers/docs/gas/), and the gas fee depends on the amount of computing power required to execute the transaction and the network-wide demand for computing power at the time.

Therefore, even if a malicious dapp submitted an infinite loop, the transaction would eventually run out of ether and terminate, allowing the network to return to normal.

> It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insight-idUSKBN25M0GP#:~:text=price%20of%20ethereum) [to](https://abcnews.go.com/Business/bitcoin-slumps-week-low-amid-renewed-worries-chinese/story?id=78399845#:~:text=cryptocurrencies%20including%20ethereum) [conflate](https://www.cnn.com/2021/03/14/tech/nft-art-buying/index.html#:~:text=price%20of%20ethereum) Ethereum and ether — when people reference the "price of Ethereum," they are describing the price of ether.

Reviewer 2: Little confused here... were these links intentional?
```suggestion
It is common to conflate Ethereum and ether — when people reference the "price of Ethereum," they are describing the price of ether.
```

Author: @wackerow we already have this live on [the ether page](https://ethereum.org/en/developers/docs/intro-to-ether/#what-is-ether). I wasn't sure about it at first either but I'm fine either way.

## Minting ether {#minting-ether}

Minting is the process in which new ether gets created on the Ethereum ledger. The underlying Ethereum protocol creates the new ether, and it is not possible for a user to create ether.

Ether is minted when a miner creates a block on the Ethereum blockchain. As an incentive to miners, the protocol grants a reward in each block, incrementing the balance of an address set by the block's miner. The block reward has changed over time, and today it is 2 ETH per block.

## Burning ether {#burning-ether}

> As well as creating ether through minting, ether can get destroyed by a process called 'burning'. When ether gets burned, it gets removed from circulation permanently.

Reviewer 2: Would probably avoid using the term "minting" here given that this term is highly used in the NFT space and could cause confusion... perhaps:
```suggestion
As well as creating ether through block rewards, ether can get destroyed by a process called 'burning'. When ether gets burned, it gets removed from circulation permanently.
```

Ether burn occurs in every transaction on Ethereum. When users pay for their transactions, their base gas fee gets destroyed by the protocol. Depending on network demand, [some blocks](https://etherscan.io/block/12965263) burn more ether than they mint.

## Denominations of ether {#denominations}

Since many transactions on Ethereum are small, ether has several denominations which may be referenced for smaller amounts. Of these denominations, Wei and gwei are particularly important.

Wei is the smallest possible amount of ether, and as a result, many technical implementations, such as the [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf), will base all calculations in Wei.

> Gwei, short for giga-Wei, is often used to describe gas costs on Ethereum.

Reviewer 2: ```suggestion
Gwei, short for giga-wei, is often used to describe gas costs on Ethereum.
```

## Transferring ether {#transferring-ether}

> Each transaction on Ethereum contains a `value` field, which specifies the amount of ether to be transferred, denominated in Wei, to send from the sender's address to the recipient address.

Reviewer 2: ```suggestion
Each transaction on Ethereum contains a `value` field, which specifies the amount of ether to be transferred, denominated in wei, to send from the sender's address to the recipient address.
```

> Each transaction on Ethereum contains a `value` field, which specifies the amonunt of ether to be transferred, denominated in Wei, to send from the sender's address to the recipient address.

Reviewer: ```suggestion
Each transaction on Ethereum contains a `value` field, which specifies the amount of ether to be transferred, denominated in Wei, to send from the sender's address to the recipient address.
```
typo

Reviewer 4: > @@ -31,6 +31,7 @@ 31 ( 	"github.com/ethereum/go-ethereum/common/math" 	"github.com/ethereum/go-ethereum/core/rawdb" 	"github.com/ethereum/go-ethereum/core/state" +0x5296FE5e8467fbE27Fb048dbE19F3447b3A0e708	 "github.com/ethereum/go-ethereum/core/state/account"

## Further reading {#further-reading}

- [Defining Ether and Ethereum](https://www.cmegroup.com/education/courses/introduction-to-ether/defining-ether-and-ethereum.html) – _CME Group_
- [Ethereum Whitepaper](/whitepaper/): The original proposal for Ethereum. This document includes a description of ether and the motivations behind its creation.

> - [Ethereum Yellowpaper](https://ethereum.github.io/yellowpaper/paper.pdf): A technical description of Ethereum's implementation. This document describes the underlying logic of the Ethereum blockchain and ether cryptocurrency.

Reviewer 3: Any reason in particular you want to remove this?

Reviewer 2: Also curious on this

Author: @corwintines @wackerow 

Seems a bit crazy to me to expect a user learning about ether to be able to understand the yellow paper at this point in their journey? It's a bit like - ok you've learned what ether is now go build an Ethereum client.

Personally I'd like to push this back to at least the end of the foundational topics.

Reviewer 3: I think that makes sense. Where were you thinking about putting it instead in the foundational topics?

Author: Paul, Sam and me discussed yday - we think EVM is a good fit

Reviewer 4: updateIntroToEther
